amarachukwu-agbo/hello-books
# Add end-to-end tests

#### What does this PR do?

Adds end-to-end tests for the application

#### Description of Task to be completed?

Test the application using Selenium and Nightwatch

#### How should this be manually tested?
Clone the repo and install application dependencies. Run command  `npm run start:server`, `npm run start:client`, `npm run e2e` on different consoles. You should the tests passing on the chrome browser on 127.0.0.0:3000.

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

#159058496

#### Screenshots (if appropriate)

N/A

#### Questions: N/A
